packages/react-table/src/components/Table/CollapseColumn.tsx

```diff
@@ -29,7 +29,9 @@ export const CollapseColumn: React.FunctionComponent<CollapseColumnProps> = ({
         onClick={onToggle}
         aria-expanded={isOpen}
       >
-        <AngleDownIcon />
+        <span className="pf-c-table__toggle-icon">
```

Comment:
Why is the `pf-c-table__toggle-icon` hardcoded?

Reply:
Replaced hard-coded className with react-styles variable.